Mathlib/Order/WithBot.lean

```diff
@@ -394,16 +400,17 @@ theorem le_coe_unbot' [Preorder α] : ∀ (a : WithBot α) (b : α), a ≤ a.unb
   | ⊥, _ => bot_le
 #align with_bot.le_coe_unbot' WithBot.le_coe_unbot'
 
-theorem unbot'_bot_le_iff [LE α] [OrderBot α] {a : WithBot α} {b : α} :
-    a.unbot' ⊥ ≤ b ↔ a ≤ b := by
-  cases a <;> simp [none_eq_bot, some_eq_coe]
-#align with_bot.unbot'_bot_le_iff WithBot.unbot'_bot_le_iff
+theorem unbot'_le_iff [LE α] {a : WithBot α} {b c : α} (h : a = ⊥ → b ≤ c) :
+    a.unbot' b ≤ c ↔ a ≤ c := by
+  cases a
+  · simpa using h rfl
+  · simp [some_eq_coe]
```

Comment:
Maybe keep the `#align` with a `ₓ`?
```suggestion
theorem unbot'_le_iff [LE α] {a : WithBot α} {b c : α} (h : a = ⊥ → b ≤ c) :
    a.unbot' b ≤ c ↔ a ≤ c := by
  cases a
  · simpa using h rfl
  · simp [some_eq_coe]
#align with_bot.unbot'_bot_le_iff WithBot.unbot'_bot_le_iffₓ
```

Reply:
I don't think the `x`s in `#align` are that useful anymore. See also https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/.60.E2.82.93.60.20in.20.60.23align.60.20.235847/near/390819190